dist/vue-strap.min.js

```diff
@@ -1,58 +0,0 @@
=n(47)(Array,"Array",function(e,t){this._t=s(e),this._i=0,this._k=t},function(){var e=this._t,t=this._k,n=this._i++;return!e||n>=e.length?(this._t=void 0,o(1)):o(0,"keys"==t?n:"values"==t?e[n]:[n,e[n]])},"values"),i.Arguments=i.Array,r("keys"),r("values"),r("entries")},function(e,t){e.exports=function(){}},function(e,t){e.exports=function(e,t){return{value:t,done:!!e}}},function(e,t,n){t.f=n(56)},function(e,t,n){e.exports={default:n(64),__esModule:!0}},function(e,t,n){n(65);var r=n(23).Object;e.exports=function(e){return r.getOwnPropertyNames(e)}},function(e,t,n){n(21)("getOwnPropertyNames",function(){return n(66).f})},function(e,t,n){var r=n(9),o=n(67).f,i={}.toString,s="object"==typeof window&&window&&Object.getOwnPropertyNames?Object.getOwnPropertyNames(window):[];e.exports.f=function(e){return s&&"[object Window]"==i.call(e)?function(e){try{return o(e)}catch(e){return s.slice()}}(e):o(r(e))}},function(e,t,n){var r=n(7),o=n(20).concat("length","prototype");t.f=Object.getOwnPropertyNames||function(e){return r(e,o)}},function(e,t){"use strict";t.__esModule=!0,t.default=function(e,t){if(!(e instanceof t))throw new TypeError("Cannot call a class as a function")}},function(e,t,n){"use strict";t.__esModule=!0;var r=function(e){return e&&e.__esModule?e:{default:e}}(n(40));t.default=function(){function e(e,t){for(var n=0;n<t.length;n++){var o=t[n];o.enumerable=o.enumerable||!1,o.configurable=!0,"value"in o&&(o.writable=!0),(0,r.default)(e,o.key,o)}}return function(t,n,r){return n&&e(t.prototype,n),r&&e(t,r),t}}()},function(e,t,n){e.exports={render:function(){var e=this.$createElement,t=this._self._c||e;return t("div",{staticClass:"hidden-print hidden-xs hidden-sm"},[t("nav",{staticClass:"bs-docs-sidebar",class:{affix:this.affixed},style:{marginTop:this.top}},[this._t("default")],2)])},staticRenderFns:[]}},function(e,t,n){(function(t){"use strict";e.exports={bind:function(e){e.dataset.isShown="true",e.style.position="relative";var n=e.getAttribute("alt")||"Expand Content",r=t('<div class="content"></div>');t(e).contents().appendTo(r),t(e).empty(),t(e).append(r),t(e).attr("class",e.className+" closeable-wrapper");var o=t('<span class="closeable-button label label-default hidden-print" style="display: none; position: absolute; top: 0; left: 0; cursor: pointer;background: #d9534f;"><span class="glyphicon glyphicon-remove" aria-hidden="true"></span></span>');t(e).append(o);var i=t('<a class="closeable-show hidden-print" style="display: none; cursor: pointer;text-decoration: underline">'+n+"</a>");t(e).append(i),o.click(function(){e.dataset.isShown="false",i.show(),o.hide(),r.get()[0].style.display="none"}),i.click(function(){e.dataset.isShown="true",i.hide(),r.get()[0].style.display=""}),t(e).on("mouseover",function(){"false"!==e.dataset.isShown&&o.show()}),t(e).on("mouseout",function(){"false"!==e.dataset.isShown&&o.hide()})}}}).call(t,n(72))},function(e,t,n){(function(t){e.exports=t.jQuery=n(73)}).call(t,function(){return this}())},function(e,t,n){var r;
```

Comment:
This file should not be deleted. :P Retaining the original `vue-strap.min.js` will do.